### ISSUE TITLE
Validation is too slow...

### ISSUE THREAD

**Issue by mariolew** [facebookresearch/PoseWarper#4]
Hi, I just run posetrack18_helper.py, but it runs too slow...... It just hangs there, I use CUDA 9.0... I wonder why it runs too slow...

**Comment by gberta** [comment]
The first thing that posetrack18_helper.py does is train the baseline model, and our posewarper model. That's going to take some time (~6-10 hours using 8 GPUs I believe). If you want to skip the training and use our pretrained models instead, you need to modify posetrack18_helper.py so that it  would skip the training stage, and use our pretrained models for inference. That way, the script will run much faster. 

Furthermore, if you want to train the models from scratch, and see a more frequent progress  of the training (otherwise you might thing that it's hanging), please change the value of PF (print frequency) in posetrack18_helper.py to a smaller value (i.e., PF = 100). Then it will print training loss after every 100 iterations.

**Comment by mariolew** [comment]
Thanks a lot.
